mentoring/components/templates/html/mcqblock.html

```diff
@@ -0,0 +1,19 @@
+<fieldset class="choices questionnaire">
+  <legend class="question">
+    {% if not hide_header %}<h3 class="question-title">{{ self.display_name }}</h3>{% endif %}
+    <p>{{ self.question }}</p>
+  </legend>
+  <div class="choices-list">
+    {% for choice in custom_choices %}
+    <div class="choice">
+      <div class="choice-result fa icon-2x"></div>
```

Comment:
@antoviaque Both of those should now be fixed with b53159dff4a4a0a51739cc940e3b9ba58609c3ed - one problem was MCQ/MRQ/Ratings created in Studio were missing a `name`. I have now added validation to flag that as an error.